YoCode/TestCountCheck.cs

```diff
@@ -51,13 +51,18 @@ public void ExecuteTheCheck()
 
         public void StoreCalculations(List<int> tempStats)
         {
-            if(tempStats != null)
+            if(tempStats.Count == 4)
             {
                 stats.totalTests = tempStats[0];
                 stats.testsPassed = tempStats[1];
                 stats.testsFailed = tempStats[2];
                 stats.testsSkipped = tempStats[3];    
-            }        
+            }
+            else
+            {
+                UnitTestEvidence.FeatureImplemented = false;
```

Comment:
Shouldn't this be `UnitTestEvidence.SetFailed("Couldn't get information about tests")?

Reply:
It should indeed. Thank you Michael